### ISSUE TITLE
`subscription_events` is out of date

### ISSUE THREAD

**Issue by mallen5311** [bufferapp/stripe-pipeline#4]
Heya @michael-erasmus and @davidgasquez ! 👋 

It appears that `buda_stripe.subscription_events` last received an event on 2018-04-25.  
https://looker.buffer.com/sql/xxkvhgxy26gsv6

This is the table used to model stripe trials (in both `dbt.stripe_trial_start_events` and `dbt.stripe_trials`).  As a result, looks that are using the `user_stripe_trials` view will not being populating after 2018-04-25
https://looker.buffer.com/explore/users/users?qid=midwqw9egfmbC4OhxJ4aPh&origin_space=86&toggle=fil,vis

**Comment by michael-erasmus** [comment]
hey @mallen5311, thank you for reporting this!!

Oops, this seems to have been caused by a configuration error, so sorry for not picking that up earlier! The service was setup to use AWS keys that were rotated on the 25th of April, which is why it broke. 

I just got things back up and running again, and seems to be pulling in data again. The service is designed to backfill missing data, starting from the most recent data and it seems like the backfill has caught up to today again!

https://looker.buffer.com/sql/7xzqmnv9qrqcyp

As a temporary fix I set it to use the new rotated keys, but as a further measure I think we'll move this to our Apollo GKE cluster cc @davidgasquez
